install/docker-compose.aws-dev.yaml

```diff
@@ -0,0 +1,60 @@
+version: '3.8'
+
+services:
+  cache:
+    build:
+      context: ./cache
+    image: wikijump/memcached
+    restart: always
+  database:
+    build:
+      context: ./postgres
+    restart: always
+    image: wikijump/postgres
```

Comment:
Important note: you're using `wikijump` as the org, but in our current docker docs we use `scpwiki`, and `scpwiki/postgres` is the one currently updated. I guess this would explain why the dev deploy currently live has an outdated seed database.

If we're going to move to `wikijump` we should aim to decommission the `scpwiki` one to prevent duplicative issues like this.

Reply:
This file is not needed at all, I will remove.